packages/node-opentelemetry/bin/opentelemetry-instrument.ts

```diff
@@ -30,7 +30,3 @@ start().catch((error) => {
   console.error(error);
   process.exit(1);
 });
-
-process.on('SIGTERM', () => {
```

Comment:
It wasn't clear to me what case this is handling. If the process gets a `SIGTERM` it should exit by default, and otherwise a handler would be attached and we should respect that handler's decision to exit or not.

Reply:
Makes sense. I think this is from the copy-n-paste